doc/rst/language/spec/modules.rst

```diff
@@ -272,10 +272,33 @@ naming (:ref:`Explicit_Naming`).
 Visibility Of A Module
 ~~~~~~~~~~~~~~~~~~~~~~
 
-A top-level module is available for use (:ref:`Using_Modules`) or
-import (:ref:`Importing_Modules`) anywhere. The visibility of a nested module is
-subject to the rules of :ref:`Visibility_Of_Symbols`, where the nested module is
-considered a "module-scope symbol" of its outer module.
+A top-level module is available for use (:ref:`Using_Modules`) or import
+(:ref:`Importing_Modules`) anywhere.  A module name is not accessible in other
+statements or expressions unless an ``import`` or ``use`` statement has brought
+the name into scope.
+
+Additionally, ``use`` and ``import`` can both name a module with a relative
+path; for example, ``this.Submodule`` or ``super.Siblingmodule``.  ``use`` and
+``import`` differ in their behavior towards a named module when two conditions
+are both true: when the named module is not a top-level module and when a
+relative path is not provided.
+
+For the purpose of ``use``, the visibility of a nested module is subject to the
+rules of :ref:`Visibility_Of_Symbols`, where the nested module is considered a
+"module-scope symbol" of its outer module.  If the module is currently in scope,
+then it may be used with just its name.  The module may alternatively be
+accessed explicitly with all the outer modules surrounding it to the top level,
+or relatively from the current scope with ``this`` or ``super`` components as
+has already been mentioned.
+
+For the purpose of ``import``, the nested module is known to be defined at the
+same location as for ``use`` statements.  However, in order to ``import`` it,
```

Comment:
I'm having trouble with this sentence ("known to be defined at"). Could we say instead

```
Now, let's consider how ``import`` interacts with nested modules. In order to ``import`` it ...
```

also change `may` to `must` in the below.

Reply:
I'm going to follow this, but use the same pluralization as the rest of the paragraph - so "interacts with a nested module" instead of "interacts with nested modules".  Also, I'm thinking `cannot` instead of `must not`?

Reply:
Sounds good